src/backend/requirements.txt

```diff
@@ -0,0 +1,40 @@
+atomicwrites==1.4.0
+attrs==20.3.0
+bidict==0.21.2
+certifi==2020.12.5
+chardet==3.0.4
+click==7.1.2
+colorama==0.4.4
+coverage==5.4
+enum34==1.1.10
+Flask==1.1.2
+Flask-Cors==3.0.10
+Flask-PyMongo==2.2.0
+Flask-SocketIO==5.0.1
+idna==2.7
+iniconfig==1.1.1
+itsdangerous==1.1.0
+Jinja2==2.11.3
+MarkupSafe==1.1.1
+mock==4.0.3
+mongomock==3.22.1
+more-itertools==8.7.0
+packaging==20.9
+pluggy==0.13.1
```

Comment:
This file is for workflows, this has all of the dependencies used by the backend